src/PredictionFramework.cpp

```diff
@@ -8,12 +11,14 @@ PredictionFramework::PredictionFramework(
     const ModelInputParameters& modelInputParameters,
     const InputObservations& observations,
     Random::RNGInterface::Sptr rng,
+    const std::string& outDir,
     Utilities::logging_stream::Sptr log)
      : model_(model),
-       seedlist_(modelInputParameters.seedlist),
-       dayShut_(modelInputParameters.day_shut),
-       modelStructure_(modelInputParameters.model_structure),
-       rng_(rng)
+       modelInputParameters_(modelInputParameters),
```

Comment:
The unused member variables (`seedlist_`, `dayShut_` etc.) are still present in the class definition. If you aren't using them in your implementation, please delete them.